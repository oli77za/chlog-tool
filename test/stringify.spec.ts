import stringify from '../src/strigify';
import {
  completeStructure,
  emptyStructure,
} from './__fixtures__/stringify.fixtures';

describe('Strigify', () => {
  it('Should stringify an empty structure', () => {
    const structure = emptyStructure;

    const result = stringify(structure);

    expect(result).toMatchSnapshot();
  });

  it('Should stringify a complete structure', () => {
    const structure = completeStructure;

    const result = stringify(structure);

    expect(result).toMatchSnapshot();
  });
});
