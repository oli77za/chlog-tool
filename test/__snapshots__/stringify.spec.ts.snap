// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Strigify Should stringify a complete structure 1`] = `
"# Changelog

All notable changes to this project will be documented in this file.

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [Unreleased]

### Added 

- Initial implementation

## [X.Y.Z] 0000-00-00

Lorem ipsum
[unreleased]: https://github.com/oli77za/chlog-tool
[X.Y.Z]: https://github.com/oli77za/chlog-tool/foo
"
`;

exports[`Strigify Should stringify an empty structure 1`] = `
"


"
`;

exports[`Strigify Should stringify and empty structure 1`] = `
"


"
`;
